gadget/layout.go

```diff
@@ -244,6 +255,20 @@ func LayoutVolumePartially(volume *Volume) (*PartiallyLaidOutVolume, error) {
 	return vol, nil
 }
 
+func setOnDiskLabelAndTypeInLaidOuts(encType secboot.EncryptionType, los []LaidOutStructure) {
```

Comment:
nitpick: usually our set function arguments are dest value, so the two inputs would be swapped

Reply:
Done now